api/api.go

```diff
@@ -17,6 +17,7 @@ type ClientService interface {
 	SyncNotify(flux.InstanceID) error
 	SyncStatus(flux.InstanceID, string) ([]string, error)
 	UpdatePolicies(flux.InstanceID, flux.PolicyUpdates) (job.ID, error)
+	LogEvent(inst flux.InstanceID, e flux.Event) error
```

Comment:
I guess we could have a daemonClient; or just include it in the DaemonService API but have a very thin wrapper around the transport, for daemon.LogEvent to use. Which may amount to the same thing.